blurb/blurb.py

```diff
@@ -1108,14 +1108,14 @@ def print(*a, sep=" "):
 git_add_files = []
 def flush_git_add_files():
     if git_add_files:
-        subprocess.run(["git", "add", "-f", *git_add_files], stdout=subprocess.PIPE, stderr=subprocess.PIPE).check_returncode()
```

Comment:
How about introducing a flag (`--verbose` CLI option or config) for controlling this? Dumping output out would probably confuse users, who are not familiar with Git.

Reply:
That wouldn’t have helped me. Since the issue seemed to happen just the once, and then not on subsequent runs, I had no opportunity to retry with verbose. The intention here is to avoid swallowing errors.

The behavior on git add is not to output anything so I don’t expect novice users to see anything different in the nominal case.